python/ray/train/lightning/lightning_trainer.py

```diff
@@ -586,7 +586,7 @@ def _lightning_train_loop_per_worker(config):
         for plugin in trainer_config.get("plugins", [])
         if not isinstance(plugin, ClusterEnvironment)
     ]
-    trainer_config["plugins"].append(RayEnvironment())
+    trainer_config["plugins"].append(RayLightningEnvironment())
 
     # Setup ddp strategy for ray orchestration
     if "strategy" in trainer_config:
```

Comment:
Would it make sense to call the new `prepare_trainer` function for these validation steps??

Reply:
done